### ISSUE TITLE
Input data format

### ISSUE THREAD

**Issue by Lila14** [guanjue/IDEAS_2018#7]
I was using the test data and noticed that the vast majority of values in the signal files are either 2 or 16. Was there some pre-processing done to this data?

**Comment by guanjue** [comment]
Yes, the signal was the -log10(p-value) based on the negative binomial model after normalized by s3norm. Since it is -log10(p-value), the 2 and 16 represent the p-value = 0.01 and p-value=1e-16.

**Comment by Lila14** [comment]
Why are these the only p-values?

**Comment by guanjue** [comment]
In the test data, we have input data for our sample, so we used p-value to take into account the local background changes of the input signal.
